Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
```

Comment:
It is a utility method to allow _ABContact to swap between WACContactDate and NSDate, so I thought this would be a good place for it -- open to suggestions if it is better in a different place

---
In reply to: [75146840](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75146840) [](ancestors = 75146840)